azure-stack/hci/manage/arc-extension-management.md

```diff
@@ -73,6 +122,19 @@ To enable an automatic upgrade, navigate to the **Extensions** page and perform
 
     :::image type="content" source="media/arc-extension-management/arc-extension-enable-auto-upgrade-2.png" alt-text="Screenshot of the notification to enable auto upgrade in the Azure portal." lightbox="media/arc-extension-management/arc-extension-enable-auto-upgrade-2.png":::
 
+# [Azure CLI](#tab/azurecli)
```

Comment:
Change to an H4 heading as mentioned given the main heading in this section is a H3 heading.